go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/juju/schema
+
+go 1.13
```

Comment:
I'm not sure if this has any implications, but I'm a little uneasy about specifying an unreleased version.

Reply:
good point. changed to go1.11.